src/main/scala/akka/actor/BeamLoggingFSM.scala

```diff
@@ -12,6 +15,8 @@ trait BeamLoggingFSM[S, D] extends FSM[S, D] { this: Actor =>
 
   private[akka] override val debugEvent = context.system.settings.FsmDebugEvent
 
+  private val debugMessages = context.system.settings.AddLoggingReceive
```

Comment:
Should we be using the built in debug messages for this? It seems to be different enough so as to not be the same thing and require it's own configuration.